### PR TITLE
feat: Add April's fact

### PR DESCRIPTION
**Pull request by sarahmonster**
Let's get facting again!

This also introduces some very small changes to the visual design, mostly to account for Nautilus' updated Link component. I think it looks sharper:

image

I had some weirdness whilst trying to update Nautilus, so I'm unclear on whether it worked or not—I got a bunch of errors, and then I had to roll back changes to my package.json in order to get the site to build. 

If the links don't match the above screenshot, something went wrong and we may need to bugfix.

### CODE REVIEW

File: src/content/facts/12-de-montfort.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Giant octopuses really aren't a thing.
+source: Wikipedia
+sourceUrl: https://en.wikipedia.org/wiki/Pierre_Denys_de_Montfort
+image: /images/colossal-octopus.jpg
+imageSource: https://en.wikipedia.org/wiki/Pierre_Denys_de_Montfort
+---
+
+Pierre Denys de Montfort was a French naturalist who had something of an obsession with giant octopuses. And you know how, when you think about something a whole lot, you start seeing it everywhere? Yep, that's what happened here.
```
Comment (tofumatt) [inline src/content/facts/12-de-montfort.md]: On first pass this sentence read a bit awkwardly to me. Here's my suggestion to tighten it up a little:

```suggestion
Pierre Denys de Montfort was a French naturalist who had something of an obsession with giant octopuses. You know how thinking about something a lot, you start seeing it everywhere? Yup, that's what happened here.
```

File: src/content/facts/12-de-montfort.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+de Montfort told everyone they'd sunk a group of warships that mysteriously disappeared one night in 1782. Turns out the disappearance was less of a mystery than originally proposed and he was summarily discredited. His career tanked (probably attacked by a giant octopus), and he died destitute.
```
Comment (tofumatt) [inline src/content/facts/12-de-montfort.md]: "They'd" is _quite_ an ambiguous noun here—I thought it meant that de Montfort told people that the people had sunk a warship. How about we clarify:

```suggestion
One night in 1782, when a group of warships mysteriously disappeared, de Montfort claimed it was the work of giant octopuses. Turns out the disappearances weren't a mystery, but were a secret kept by the British Navy. de Montfort was discredited, his career tanked (probably attacked by a giant octopus), and he died destitute.
```

**Review by tofumatt — CHANGES_REQUESTED** [review]
I have some thoughts on the copy text—let me know what you think.

**Comment by tofumatt** [comment]
(Love the image, by the way.)

**Comment by tofumatt** [comment]
(This wasn't building for me locally but an update to the dependencies got things working for me.)

**Comment by sarahmonster** [comment]
Okay! I started with your copyedits, and then they got more copyedity as I kept going. I've also updated Nautilus to pick up recent changes to links, and I've fleshed out the README a wee bit. 

Let me know what you think!

**Review by tofumatt — APPROVED** [review]

**Pull request merged**